taxiunn/driver_auth/apps.py

```diff
@@ -0,0 +1,8 @@
+from django.apps import AppConfig
+
+
+class DriverAuthConfig(AppConfig):
+    """Определение конфигурации Django-приложения."""
+
+    default_auto_field = 'django.db.models.BigAutoField'
+    name = 'driver_auth'
```

Comment:
лучше везде указывать типы
```suggestion
    default_auto_field: str = 'django.db.models.BigAutoField'
    name: str = 'driver_auth'
```

Reply:
это генерируемые django файлы